src/main/resources/db/migration/V3__create_salesemp_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE salesEmployees (
+    salesEmployeeId int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    firstName varchar(50) NOT NULL,
+    lastName varchar(50) NOT NULL,
+    salary decimal(10,2) NOT NULL,
+    currency char(3) NOT NULL,
+    bankAccountNo varchar(34) NOT NULL,
+    nationalInsuranceNo char(9)
```

Comment:
blank nationalInsuranceNumber? Seems like a security risk

Reply:
"If you find a job before having your NIN, you can be hired as long as you are qualified to work in the UK without a working permit, that is i.e, if you have a European passport."

https://www.smallworldfs.com/en/blog/how-to-work-without-a-nin-in-the-uk